pkg/cli/tsdump_upload.go

```diff
@@ -226,7 +227,7 @@ func (d *datadogWriter) emitDataDogMetrics(data []DatadogSeries) error {
 	}
 
 	fmt.Printf(
-		"tsdump datadog upload: sending payload containing %d series including %s\n",
+		"\033[G\033[K tsdump datadog upload: uploading metrics containing %d series including %s",
```

Comment:
In the debug zip upload process, I'm logging the progress on STDERR. And logging the uploadID on STDOUT.
Because the upload ID is a useful bit of information that can be further chained with another command or reused through some other automation. So, segregating "logs"/"progress" and "output" can be useful. 

For example, if the user does something like this:

```bash
cockroach debug tsdump ... | awk <get the upload id> | xargs <do something else>
```
The upload ID (being part of STDOUT) will be fed into the next command `awk`. But the progress/other logs (on STDERR) will continue to be printed on the screen.

I've seen this patten in multiple CLI tools.